setup.cfg

```diff
@@ -0,0 +1,38 @@
+[metadata]
+name = i-PI
+version = 2.3
+description = A Python interface for ab initio path integral molecular dynamics simulations
+long_description = file: README.rst
+long_description_content_type = text/x-rst
+author = Michele Ceriotti
```

Comment:
Maybe the authors could be ammended :-) i-PI development team and the e-mail of i-PI coordinators?

Reply:
Sure, would that be ipi.managers@gmail.com?

Reply:
yes (note to self we need to add more people to this alias)